chainservice/chainservice.go

```diff
@@ -105,9 +108,37 @@ func (cs *ChainService) HandleAction(ctx context.Context, actPb *iotextypes.Acti
 	if err != nil {
 		log.L().Debug(err.Error())
 	}
+	hash, err := act.Hash()
+	if err != nil {
+		return err
+	}
+	cs.actionsync.ReceiveAction(ctx, hash)
```

Comment:
only do this for unicast?

Reply:
add a TODO, will fix after tx type introduced